apps/src/templates/manageStudents/manageStudentsRedux.js

```diff
@@ -31,13 +35,33 @@ export default function manageStudents(state=initialState, action) {
       studentData: action.studentData,
     };
   }
+  if (action.type === START_EDITING_STUDENT) {
+    let updatedStudentData = state.studentData;
+    updatedStudentData[action.studentId].isEditing = true;
```

Comment:
This is mutating existing state (updatedStudentData is a reference to state.studentData, not a copy of it)